templates/pre-commit/requirements.in

```diff
@@ -0,0 +1,8 @@
+pip-tools
+pre-commit
+
+# Example hooks
+pre-commit-hooks
```

Comment:
should these be commented out or something?

Reply:
No